scripts/microrepl.py

```diff
@@ -48,12 +48,10 @@ def connect_miniterm(port):
                 "then try again.\n"
             )
         elif e.errno == errno.EACCES:
-            print("Found the device, but could not connect.".format(port),
-                  file=sys.stderr)
-            print(e, file=sys.stderr)
-            print('On linux, try adding yourself to the "dialout" group',
-                  file=sys.stderr)
-            print('sudo usermod -a -G dialout <your-username>', file=sys.stderr)
+            sys.stderr.write("Found the device, but could not connect.".format(port))
```

Comment:
Have you experienced any problems due to `print(..., file=sys.stderr)` instead of `sys.stderr.write(...)`?

Reply:
yes, got some error messages under macOS Sierra 10.12.6 and Python 3.5.1